doc/DeveloperGuide.md

```diff
@@ -60,7 +60,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | create groups of contacts | better organize my contacts
```

Comment:
Can be a bit more specific here. Maybe `user who is involved in many social circles`?